libethereum/Executive.cpp

```diff
@@ -127,14 +126,10 @@ void StandardTrace::operator()(uint64_t _steps, Instruction inst, bigint newMemS
 		r["storage"] = storage;
 	}
 
-	if (returned || newContext)
```

Comment:
We made several changes in these files during the last weeks - did you actually intend to do these changes apart from the PC change?